tests/strings/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -191,48 +191,13 @@ const Bar = styled.div\`
 
 =====================================output=====================================
 foo(
-  \`a long string \${1 +
-    2 +
-    3 +
-    2 +
-    3 +
-    2 +
-    3 +
-    2 +
-    3 +
-    2 +
-    3 +
-    2 +
-    3 +
-    2 +
-    3 +
-    2 +
-    3} with expr\`
+  \`a long string \${
+    1 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + 3
+  } with expr\`
 );
 
-const x = \`a long string \${1 +
-  2 +
-  3 +
-  2 +
-  3 +
-  2 +
-  3 +
-  2 +
-  3 +
-  2 +
-  3 +
-  2 +
-  (function() {
-    return 3;
-  })() +
-  3 +
-  2 +
-  3 +
-  2 +
-  3} with expr\`;
-
-foo(
-  \`a long string \${1 +
+const x = \`a long string \${
+  1 +
     2 +
     3 +
```

Comment:
Would be better if the expression were wrapped like in `if`:

**Prettier pr-7010**
[Playground link](https://deploy-preview-7010--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEAbOMAEBDTBeTAAwBJhMAjASwDMIAnKAYTsqgHM46BZCAZwAs4AEwAKnAA7oAHgGVsAdyGdMAahy8A1gBU6EAG4JsAITiGAInHEIhvAEoR52KL1WY2qSgFtecAOKoAT3F+XgAJbAAvbDohADkICHFeLUo4FLhMAF9CAG4AHSgCmkwACipaBmZWDm4+QVEJaTlFZTVsTR19QxNzS2s7BycXNXcvH38gkPComPjE5NT0gEpMYAKCzJAAGhBEmEpoXmRQaN15EWiEI5RsPQhKIW2QcjpsMA0MGXE36uQYOgArnAdvwYJ5UAB1fiUeC8b5gOAyK4wyh6GEBZDgXhHHasHx0GAiV5sTzYZDUbCoHw7ABWvCkRle70+2E8cAAMqw4OTKdSQHTZNV0ABFAEQeA8qnAkDfOj4zHkbDkOCoJ7iFiwCEPGD8ZAADgADDt1Xw4BDXuJMeq4PiDE8AI5i+BE+aY9oAWigpiUjx2dDgjso-qJ2BJZKQFKlOx8nkof0B0t4QrgovF3IjvOlMCVWqEOuQACYdv9sJQPOxGBBPKTMVBoNydgCfFolddI3y9ECAJJQJSwGRgFjiGAAQV7MhgAXQkp8mUyQA)
```sh
--parser babel
```

**Input:**
```jsx
let a = `${ bifornCringerMoshedPerplexSawder + askTrovenaBeenaDependsRowans + glimseGlyphsHazardNoopsTieTie }`;

if (bifornCringerMoshedPerplexSawder + askTrovenaBeenaDependsRowans + glimseGlyphsHazardNoopsTieTie) {

}
```

**Output:**
```jsx
let a = `${
  bifornCringerMoshedPerplexSawder +
    askTrovenaBeenaDependsRowans +
    glimseGlyphsHazardNoopsTieTie
}`;

if (
  bifornCringerMoshedPerplexSawder +
  askTrovenaBeenaDependsRowans +
  glimseGlyphsHazardNoopsTieTie
) {
}

```